File: programmers/yuhwang/12949.swift
import Foundation

func solution(_ arr1:[[Int]], _ arr2:[[Int]]) -> [[Int]] {
    var answer = Array(repeating: Array(repeating: 0, count: arr2[0].count), count: arr1.count)
    for i in 0..<arr1.count {
        for j in 0..<arr2[0].count {
            for k in 0..<arr2.count {
                answer[i][j] += arr1[i][k] * arr2[k][j]
            }
        }
    }
    return answer
}

File: programmers/jihwjeon/12949.py
def solution(arr1, arr2):
    answer = []
    
    n = len(arr2[0])
    for row1 in arr1:
        tmp = []
        for i in range(n):
            s = 0
            for j in range(len(row1)):
                s += row1[j] * arr2[j][i]
            tmp.append(s)
        answer.append(tmp)
                
    return answer
